fakestorage/object.go

```diff
@@ -809,6 +809,9 @@ func (s *Server) downloadObject(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Length", strconv.FormatInt(contentLength, 10))
 	w.Header().Set("X-Goog-Generation", strconv.FormatInt(obj.Generation, 10))
 	w.Header().Set("X-Goog-Hash", fmt.Sprintf("crc32c=%s,md5=%s", obj.Crc32c, obj.Md5Hash))
+	for name, element := range obj.Metadata {
+		w.Header().Set("X-Goog-Meta-"+name, element)
+	}
 	w.Header().Set("Last-Modified", obj.Updated.Format(http.TimeFormat))
```

Comment:
nit: let's use `value`for the variable name and move the for loop to the end, just to make the code easier to read
```suggestion
	w.Header().Set("Last-Modified", obj.Updated.Format(http.TimeFormat))
	for name, value := range obj.Metadata {
		w.Header().Set("X-Goog-Meta-"+name, value)
	}
```